resources/views/profile/update-password-form.blade.php

```diff
@@ -1,7 +1,14 @@
 <div x-data="{isTyping: false}">
     <div class="flex flex-col">
-        <span class="text-2xl font-semibold text-theme-secondary-900">@lang('fortify::pages.user-settings.password_information_title')</span>
-        <span>@lang('fortify::forms.update-password.requirements_notice')</span>
+        <div class="my-8">
+            <x-ark-alert type="warning">
+                <x-slot name="message">
```

Comment:
can pass it as a prop `:message="trans(..)"` instead of using a slot if it's just a single line of text